dataactvalidator/validation_handlers/validationManager.py

```diff
@@ -413,6 +415,8 @@ def runValidation(self, jobId, interfaces):
             # Mark validation as finished in job tracker
             jobTracker.markJobStatus(jobId,"finished")
             errorInterface.writeAllRowErrors(jobId)
+            # Update error info for submission
+            jobTracker.getSubmissionErrorInfo(submissionId)
```

Comment:
If I'm interpreting this correctly, it looks like this code will update a submission's error/warning count after each step of the validation process occurs. That is cool...wouldn't have thought about it w/ this level of detail.